test/config/mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
+ignore_missing_imports = True
```

Comment:
what missing imports did we need this for?

Reply:
The docker python package is not found.

Reply:
note: I don't have time to try to see if we can fix mypath now. I'd suggest merging this to be back to green and see if we can track that as a ticket?